test/PermissionedExchange.test.ts

```diff
@@ -112,10 +112,10 @@ describe('PermissionedExchange Contract', () => {
             await queryRegistry.connect(indexer).updateIndexingStatusToReady(DEPLOYMENT_ID);
             // create plan template
             await planManager.createPlanTemplate(time.duration.days(3).toString(), 1000, 100, METADATA_HASH);
-            // default plan -> planId: 0
-            await planManager.connect(indexer).createPlan(etherParse('10'), 0, constants.ZERO_BYTES32);
```

Comment:
Also need to keep the test for `default plan`

Reply:
fixed